.rubocop.yml

```diff
@@ -46,7 +46,7 @@ Metrics/AbcSize:
   Max: 391
 
 Metrics/ClassLength:
-  Max: 235
```

Comment:
Really depends on the system owners view - expanding this rule for all classes may not be desirable. I'm ok either way.